ansible/roles/mail/handlers/main.yml

```diff
@@ -0,0 +1,52 @@
+# SPDX-License-Identifier: MIT
+- name: systemctl daemon-reload
+  systemd:
+    daemon_reload: yes
+- name: systemctl reload apache2
+  systemd:
+    name: apache2
+    state: reloaded
+- name: systemctl reload postgresql
+  systemd:
+    name: apache2
+    state: reloaded
```

Comment:
This is overly repetive. Can this be looped somehow?

Reply:
No, it cannot be looped. The handlers are called by the tasks if the execution of the task lead to a change on the remote host.